recipes/xapian/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xapian" %}
```

Comment:
And this would be called python-xapian

Reply:
My main idea here was to make the Python name as simple as possible, since I reckon that would be the 99 % use case in conda.

But I don't mind changing it to `python-xapian`. Is that how people tend to name bindings to non-native Python packages?

Reply:
Yeah, I think `python-xapian` is right here, especially with the new [conda namespaces](https://gist.github.com/mcg1969/da5aec380d2ed083b79ddcf151ca16f1) coming, where `xapian` will be automatically resolved to `python-xapian` if you are in a python context.  Think of this change as being forwards compatible :)